src/inbox/mod_inbox_muc.erl

```diff
@@ -93,29 +93,29 @@ direction(From, To) ->
       Packet :: packet().
 handle_outgoing_message(Host, Room, To, Packet) ->
     maybe_reset_unread_count(Host, To, Room, Packet),
-    TS = erlang:system_time(microsecond),
-    maybe_write_to_inbox(Host, To, Room, Packet, TS, fun write_to_sender_inbox/5).
+    Acc = mongoose_acc:new(#{location => ?LOCATION, lserver => Host, host_type => undefined}),
```

Comment:
Actually in https://github.com/esl/MongooseIM/pull/3097 I am trying to ensure the consistency between `lserver` and `host_type` in accumulators. For me this is against the principle of least surprise, hopefully we could rework this code later to accept `Acc` as one of its arguments.

Reply:
Makes sense, but muc code is a bit unfortunate to not really use accs, so I think it would need quite a complex muc rework, then it would all be much better 😕